_data/ph_authors.yml

```diff
@@ -2292,32 +2292,32 @@
     pt: |
        Isabelle Gribomont é pesquisadora em Humanidades Digitais na universidade de Louvain e na Biblioteca Real da Bélgica.
 
-- name: Nicolás Llano
+- name: Nicolás Llano Linares
   email: llano.nicolas@gmail.com
   twitter: enetreseles
   github: nllano
   team: true
   team_start: 2021
-  institution: Instituto Brasileiro de Ensino, Desenvolvimento e Pesquisa
+  institution: Fundação Getúlio Vargas
   sortname: Llano
   affiliation:
     en: |
-      Brazilian Institute of Education, Development and Research, Brazil
+      School of Communication, Media and Information, Fundação Getúlio Vargas, Brazil
     es: |
-      Instituto Brasilero de Enseñanza, Desarrollo e Investigación, Brasil
+      Escuela de Comunicación, Medios e Información, Fundação Getúlio Vargas, Brasil
     fr: |
-      Institut Brésilien d'enseignement, de développement et de recherche, Brésil
+      l’École de la communication, des médias et de l’information, Fundação Getúlio Vargas, Brésil
```

Comment:
remove "l'" : 
"École de la communication, des médias et de l’information, Fundação Getúlio Vargas, Brésil"

Reply:
Thank you, @marie-flesch. Done.